### DISCUSSION TITLE
"# $schema" in yaml files should stay on top (like <xml..> in XML files

### DISCUSSION THREAD

**Discussion by eitzenbe** [mathieucarbou/license-maven-plugin#886] [General & Support]
**Describe the feature**
subject says it all

**Comment by eitzenbe** [comment]
the same goes for shebangs in shell files

**Comment by mathieucarbou** [comment]
not sure this is spec-ed and not sure also this is always in the first line.

if your use--case i s such, you can define a new mapping like it is done for xml by skipping this first line.

will move to a discussion.
